portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+</head>
+
+<body>
+
+<ul class="nav_bar">
+  <div id="homepage">
+    <li class="nav_item"><a href="./index.html">Home</a></li>
+  </div>
+
+   <div id="about">
+    <li class="nav_item"><a class="active" href="./about.html">About Me</a></li>
+   </div>
+
+   <div id="experience">
+    <li class="nav_item"><a href="./experience.html">Experience</a></li>   
+   </div>
+</ul>
+
+  <body>
+      <div id="content">
+      <p class="intro">
```

Comment:
Fix formatting so that it is like
  <div>
    <p>
    </p>
  </div>

Reply:
Fixed (shown in later commit)